x11-libs/pango/pango-1.50.14.recipe

```diff
@@ -100,18 +110,20 @@ BUILD()
 
 INSTALL()
 {
-	ninja install -C build
+	ninja -C build install
 
 	prepareInstalledDevelLibs libpango-1.0 \
-		libpangocairo-1.0 libpangoft2-1.0
+		libpangocairo-1.0 \
+		libpangoft2-1.0
 	fixPkgconfig
 
 	# devel package
 	packageEntries devel \
-		$developDir
+		$developDir \
+		$dataDir/gir-1.0
```

Comment:
*.gir files should be included in the develop package otherwise build will fail at one point not finding these.